pkg/instancetype/instancetype.go

```diff
@@ -849,7 +849,8 @@ func (m *InstancetypeMethods) inferDefaultsFromDataVolume(vm *virtv1.VirtualMach
 			if dvt.Name != dvName {
 				continue
 			}
-			return m.inferDefaultsFromDataVolumeSpec(&dvt.Spec, defaultNameLabel, defaultKindLabel, vm.Namespace)
+			dvtSpec := dvt.Spec
```

Comment:
Any reason for this change?

Reply:
It was causing the following lint error after the update of golangci-lint 

```
 pkg/instancetype/instancetype.go:852:45: G601: Implicit memory aliasing in for loop. (gosec)
			return m.inferDefaultsFromDataVolumeSpec(&dvt.Spec, defaultNameLabel, defaultKindLabel, vm.Namespace) 
```
source: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/10914/pull-kubevirt-code-lint/1738136816302690304